.github/workflows/test.yml

```diff
@@ -145,14 +146,22 @@ jobs:
           sudo cpanm -n Neovim::Ext || cat "$HOME/.cpanm/build.log"
           perl -W -e 'use Neovim::Ext; print $Neovim::Ext::VERSION'
 
+      - if: ${{ matrix.build.flavor == 'release' }}
+        name: Setup Release build type
+        run: echo CMAKE_BUILD_TYPE=Release >> $GITHUB_ENV
```

Comment:
Wut? Does adding `CMAKE_BUILD_TYPE=Release` to flags not work?

Reply:
I checked locally and I can't reproduce. Have you tested it out?

Reply:
We need to fix that then, this is pretty hacky. We can use this workaround for now in any case.

Reply:
We can touch the respective files earlier in the workflow. The root of the problem is we're using cmake to build and make to test.

Reply:
Yes, that was mostly to save a ci step since we can't run the oldtests with cmake. I guess it's better to just run tests directly with cmake instead anyway?